internal/version/version_test.go

```diff
@@ -0,0 +1,52 @@
+package version
+
+import (
+	"bytes"
+	"os/exec"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestTag(t *testing.T) {
+	out, err := exec.Command("git", "show", "-s", "--format=%ct", "HEAD", Tag).CombinedOutput()
+	if err != nil {
+		if strings.Contains(err.Error(), "file not found") {
```

Comment:
An `exec.LookPath` might be a better way of dealing with this case.

Reply:
You're right, it's more correct too.